lib/msal-core/src/telemetry/TelemetryUtils.ts

```diff
@@ -1,15 +1,14 @@
-import { B2cAuthority } from "../authority/B2cAuthority";
-import { AADTrustedHostList } from "../utils/Constants";
 import { TENANT_PLACEHOLDER, EVENT_NAME_PREFIX } from "./TelemetryConstants";
 import { CryptoUtils } from "../utils/CryptoUtils";
 import { UrlUtils } from "../utils/UrlUtils";
+import { TrustedAuthority } from "../authority/TrustedAuthority";
 
 export const scrubTenantFromUri = (uri: string): String => {
```

Comment:
Can we create a utility class with static functions instead of exporting function constants?

Reply:
I think that's reasonable but since this PR isn't really changing any telemetry files other than to fix compilation, I think that can be taken separately.